px-data-grid-filter-entity.html

```diff
@@ -126,6 +126,16 @@
 
             _selectedColumnId: {
               type: String
+            },
+
+            _minNumberBound: {
+              type: Number,
+              value: 0
+            },
+
+            _maxNumberBound: {
+              type: Number,
+              value: 100
```

Comment:
Same as with 0, I think we shouldn't have any defaults.

Reply:
Not really sure if these are blockers,  but we should never end up using range selection if we don't know both min and max. So case where one is missing, we should just handle it like normal number filter (with operations).